### PR TITLE
minor debug build tweaks

### PR DESCRIPTION
**Pull request by sezero**

### CODE REVIEW

File: Makefile.in [hunk 1]
```diff
@@ -9,6 +9,7 @@ COMPAT_VERSION	= $(VERSION_MAJOR)
 
 CC		= @CC@
 CFLAGS		= -c @CFLAGS@ @DEFS@ -D_REENTRANT
+#CFLAGS += -DDEBUG
```
Comment (AliceLR) [inline Makefile.in:12]: This seems like it should be indented with two tabs.

File: lite/Makefile.in.in [hunk 2]
```diff
@@ -6,6 +6,7 @@ COMPAT_VERSION	= $(VERSION_MAJOR)
 
 CC		= @CC@
 CFLAGS		= -c @CFLAGS@ @DEFS@ -D_REENTRANT -DLIBXMP_CORE_PLAYER
+#CFLAGS += -DDEBUG
```
Comment (AliceLR) [inline lite/Makefile.in.in:9]: Same...

File: lite/Makefile.vc.in [hunk 3]
```diff
@@ -4,6 +4,7 @@
 CC	= cl
 CFLAGS	= /O2 /W3 /MD /Iinclude\libxmp-lite /DBUILDING_DLL /DWIN32 \
 	  /D_USE_MATH_DEFINES /D_CRT_SECURE_NO_WARNINGS /DLIBXMP_CORE_PLAYER
+#CFLAGS = $(CFLAGS) /DDEBUG
```
Comment (AliceLR) [inline lite/Makefile.vc.in:7]: Same...

File: src/common.h [hunk 4]
```diff
@@ -8,7 +8,7 @@
 #ifndef LIBXMP_NO_DEPACKERS
 #define LIBXMP_NO_DEPACKERS
 #endif
-#endif
+#endif /**/
```
Comment (AliceLR) [inline src/common.h:11]: ??

**Comment by AliceLR** [comment]
This mostly looks OK to me, it just looks like some spaces were accidentally committed instead of tabs in a couple places.

File: lite/Makefile.in.in (hunk 2 above)
Comment (sezero) [reply lite/Makefile.in.in]: fixed

File: Makefile.in (hunk 1 above)
Comment (sezero) [reply Makefile.in]: fixed

File: lite/Makefile.vc.in (hunk 3 above)
Comment (sezero) [reply lite/Makefile.vc.in]: fixed

File: src/common.h (hunk 4 above)
Comment (sezero) [reply src/common.h]: ugh. removed.

**Comment by sezero** [comment]
Done all the suggestions.

**Review by AliceLR — APPROVED** [review]
Thanks!

**Pull request merged**